docs/JOSS/paper.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'pyhf: pure-Python implementation of HistFactory statistical models'
+tags:
+  - Python
+  - physics
+  - high energy physics
+  - statistical modeling
+  - fitting
+  - auto-differentiation
+authors:
+  - name: Lukas Heinrich
+    orcid: 0000-0002-4048-7584
+    affiliation: 1
+  - name: Matthew Feickert^[Corresponding author.]
+    orcid: 0000-0003-4124-7862
+    affiliation: 2
+  - name: Giordon Stark
+    orcid: 0000-0001-6616-3433
+    affiliation: 3
+  - name: Kyle Cranmer
+    orcid: 0000-0002-5769-7094
+    affiliation: 4
+affiliations:
+ - name: CERN
+   index: 1
+ - name: University of Illinois at Urbana-Champaign
+   index: 2
+ - name: SCIPP, University of California, Santa Cruz
+   index: 3
+ - name: New York University
+   index: 4
+date: 5 October 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+Measurements in High Energy Physics (HEP) rely on determining the compatibility of observed collision events with theoretical predictions.
+The relationship between them is often formalised in a statistical model $f(\mathbf{x}|\mathbf{\phi})$ describing the probability of data $\mathbf{x}$ given model parameters $\mathbf{\phi}$.
+Given observed data, the likelihood $\mathcal{L}(\mathbf{\phi})$ then serves as the basis to test hypotheses on the parameters $\mathbf{\phi}$.
```

Comment:
suggest: ".... serves as the basis to for inference on the parameters $\mathbf{\phi}$."